parquet/src/file/statistics.rs

```
@@ -186,14 +186,18 @@ pub fn from_thrift(
                     // INT96 statistics may not be correct, because comparison is signed
                     // byte-wise, not actual timestamps. It is recommended to ignore
                     // min/max statistics for INT96 columns.
-                    let min = min.map(|data| {
+                    let min = if let Some(data) = min {
```

If we make invalid statistics return an error I think it will mean the reader will not be able to read any data from the file

What do you think about ignoring the error. `.ok()` style instead here -- that way the statistics would come in as `None` but the file could still be read